src/components/common/Modal/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Button,
+  IconButton,
+  Modal as ModalContainer,
+  Text
+} from '@offer-ui/react'
+import type { ReactElement, ReactNode } from 'react'
+import Image from 'next/image'
+import styled from '@emotion/styled'
+
+export interface ModalProps {
+  isOpen?: boolean
+  height?: string
+  hasLogo?: boolean
+  title: string
+  description?: string
+  primaryButtonText?: string
+  ghostButtonText?: string
+  children?: ReactNode
+  onClose(): void
+}
+
+export const Modal = ({
+  isOpen = false,
+  onClose,
+  hasLogo = false,
+  title,
+  description = '',
+  primaryButtonText = '',
+  ghostButtonText = '',
+  children
+}: ModalProps): ReactElement => {
+  return (
+    <StyledModalContainer isOpen={isOpen} onClose={onClose}>
+      <StyledTopWrapper>
+        {hasLogo && (
+          <Image alt="logo" height={18} src="/images/logo.svg" width={70} />
+        )}
+        <StyledCloseButton colorType="gray30" icon="close" onClick={onClose} />
+      </StyledTopWrapper>
+      <StyledContentWrapper hasLogo={hasLogo}>
+        <Text styleType="headline01B" tag="p">
+          {title}
+        </Text>
+        <StyledDescription styleType="body01R" tag="p">
+          {description}
+        </StyledDescription>
+      </StyledContentWrapper>
+      {children}
```

Comment:
```suggestion
      {children}
      </StyledContentWrapper>
```
children을 요기에 포함하는 건 별로일까요?.?

Reply:
요거는 UI가 대부분 타이틀과 설명이 한묶음이고 그 밑에 추가 컨텐츠가 들어가는 경우엔 마진이 들어가더라구요 !
그래서 저 두개만을 한 묶음으로 했습니다 (StyledContentWrapper 탑, 바텀에 마진값이 들어가 있어요)!